frontend/web/micro-ui-internals/packages/modules/PublicServices/src/utils/createUtils.js

```
@@ -0,0 +1,172 @@
+          ],
+          identifiers: null,
+          skills: [
+              {
+                  "type": "DRIVING",
+                  "level": "UNSKILLED"
+              }
+          ],
+          "photograph": null,
+          additionalFields: {
+              "fields": [
+                  ...data?.additionalDetails,
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)